.jenkinsci/github-api.groovy

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env groovy
+
+def mergePullRequest() {
+	if ( ! ( checkMergeAcceptance() ) ) { return false  }
+	withCredentials([string(credentialsId: 'jenkins-integration-test', variable: 'sorabot')]) {
+		def slurper = new groovy.json.JsonSlurperClassic()
+		def commitTitle = ""
+		def commitMessage = ""
+		def mergeMethod = getMergeMethod()
+		def jsonResponseMerge = sh(script: """
+		curl -H "Authorization: token ${sorabot}" \
+				 -H "Accept: application/vnd.github.v3+json" \
+				 -X PUT --data '{"commit_title":"${commitTitle}","commit_message":"${commitMessage}","sha":"${env.GIT_COMMIT}","merge_method":"${mergeMethod}"}' \
+				 -w "%{http_code}" https://api.github.com/repos/hyperledger/iroha/pulls/${CHANGE_ID}/merge""", returnStdout: true)
+		def githubResponce = sh(script:"""set +x; printf '%s\n' "${jsonResponseMerge}" | tail -n 1; set -x""", returnStdout: true).trim()
+		if ( ! ( githubResponce ==~ "200" ) ) {
+			return false
+		}
+		jsonResponseMerge = slurper.parseText(jsonResponseMerge)
+		if (jsonResponseMerge.merged != "true") {
+			return false
+		}
+		return true
+	}
+}
+
+def checkMergeAcceptance() {
+	def approvalsRequired = 2
```

Comment:
@bakhtin GitHub doesn't have an API that checks mergeability.
Moreover I didn't try to create and merge PR to protected branch on GitHub. Yes actually API method merge PR returns message in JSON. We can try to mute `checkMergeAcceptance()` method and merge this PR (when it will be finished). 
However also there is a possible issue: `iroha` repo admins are able to force merge without all checks passed. That's why I'll leave this logic unchanged